README.md

```diff
@@ -18,10 +18,16 @@ To get started with Asimov, clone the repository or download and extract an arch
 $ git clone git@github.com:stevegrunwell/asimov.git
 ```
 
-After you've cloned the repository, run the `install.sh` script to automatically:
-* Symlink Asimov to `/usr/local/bin`, making it readily available from anywhere.
-* Schedule Asimov to run once a day, ensuring new projects' dependencies are quickly excluded from Time Machine backups.
-* Run Asimov for the first time, finding all current project dependencies adding them to Time Machine's exclusion list.
+After you've cloned the repository :
+* [optional] before installing, you can edit `com.stevegrunwell.asimov.plist` to change the scheduling of Asimov (default to once a day)
```

Comment:
A bit of editing here:

* "[Optional]" should be capitalized
* Sentences should end in appropriate punctuation
* "Run `make` to automatically:"